OMCompiler/SimulationRuntime/c/simulation/solver/linearSolverKlu.h

```diff
@@ -38,8 +38,8 @@
 #define _LINEARSOLVERKLU_H_
 
 #include "../../simulation_data.h"
-#include "suitesparse/Include/amd.h"
-#include "suitesparse/Include/klu.h"
+#include "amd.h"
```

Comment:
"" kind of includes are usually indicate the header is part of you local project or code. If headers are from an outside library through an include directive you can use <> to signify that. This is of course just convention and only helps for overlook feel at a glance.

Reply:
Yeah, you are right. I changed it at some places and will do it here as well.